src/OSPSuite.Assets/UIConstants.cs

```diff
@@ -2152,6 +2152,7 @@ public static class Warning
       public static string LargeNumberOfOutputPoints(int numberOfPoints) =>
          $"The selected output resolution will generate {numberOfPoints} points and may severely impact the software performance.\nAre you sure you want to run with these setting? If not, consider changing output resolution in simulations settings";
 
+      public static string NeighborhoodWasNotFoundInModel(string neighborhoodName, string buildingBlockName) => $"The neighborhood '{neighborhoodName}' from building block '{buildingBlockName}' was not added to the simulation";
```

Comment:
Also add the module name to the message?

Reply:
It's part of the `buildingBlock.DisplayName` property which is how this is called.